### PR TITLE
dev-python/git-python: version bump

### PR DESCRIPTION
**Pull request by TheChymera**
Package-Manager: portage-2.3.0_rc1

### CODE REVIEW

**Comment by TheChymera** [comment]
So I guess all these failures are not actually related to the PR?

**Comment by mgorny** [comment]
@TheChymera yes, sorry about that. We have a Gentoo developer who committed them all and refuses to fix them in time.

The PR looks good.

**Comment by mgorny** [comment]
@gentoo/python

File: dev-python/git-python/git-python-2.0.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```
Comment (SoapGentoo) [inline dev-python/git-python/git-python-2.0.5.ebuild]: why no `EAPI=6`?

**Comment by TheChymera** [comment]
@SoapGentoo updated. Can you pull now?

**Comment by SoapGentoo** [comment]
@TheChymera as usual, please squash all your commits into one. I can pull it then.

**Pull request closed without merging**